### PR TITLE
CI: add s3fs

### PR DESCRIPTION
**Pull request by fedordikarev**
Add `gcsfuse` to `grafana-ci-deploy` docker image for further usage for publishing packages.

As a side effect, upgrade `aptly`, `google-cloud-sdk` to the actual versions.

### CODE REVIEW

File: scripts/build/ci-deploy/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM debian:testing-20210111-slim
```
Comment (malcolmholmes) [inline scripts/build/ci-deploy/Dockerfile:1]: I would rather this be a separate PR. Could have other implications for our builds in general.

File: scripts/build/ci-deploy/Dockerfile [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+FROM debian:bullseye-slim
 
 # Use ARG so as not to persist environment variable in image
-ARG GOVERSION=1.17.8 \
-  GO_CHECKSUM=980e65a863377e69fd9b67df9d8395fd8e93858e7a24c9f55803421e453f4f99 \
-  DEBIAN_FRONTEND=noninteractive
```
Comment (malcolmholmes) [inline scripts/build/ci-deploy/Dockerfile:6]: Why do you need to upgrade Golang? This would amount to a change across the whole of Grafana. Again, something that would require a separate PR.

File: scripts/build/ci-deploy/Dockerfile [hunk 3]
```diff
@@ -15,30 +15,32 @@ RUN curl -fLO https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.t
...
 # Need procps for pkill utility, which is used by the build pipeline tool to restart the GPG agent
 RUN apt update && apt install -yq curl python3-pip procps && pip3 install -U awscli crcmod && \
-    curl -fLO https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
-    echo "${GOOGLE_SDK_CHECKSUM} google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz" | sha256sum --check --status && \
-    tar xzf google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz -C /opt && \
-    rm google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
+    curl -fLO https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
+    echo "${GOOGLE_SDK_CHECKSUM} google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz" | sha256sum --check --status && \
+    tar xzf google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz -C /opt && \
+    rm google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
```
Comment (malcolmholmes) [inline scripts/build/ci-deploy/Dockerfile:33]: Is this `s/sdk/cli/` change something that Google has changed, or are we downloading something different here?

File: scripts/build/ci-deploy/Dockerfile (hunk 1 above)
Comment (fedordikarev) [reply scripts/build/ci-deploy/Dockerfile:1]: The main reason for that was that with `debian:testing-20210111` it can't build and install awscli: some problems building PyYAML.
Seems it's related to https://github.com/pypa/pip/issues/11070

I did some more investigation and found that having `pip3 install -U pip==20.0.2` could fix that.

I will update that PR to have more conservative changes.

File: scripts/build/ci-deploy/Dockerfile (hunk 2 above)
Comment (fedordikarev) [reply scripts/build/ci-deploy/Dockerfile:6]: I saw that golang used here only to build aptly and checked that it works with last version of golang.

I will create more conservative changes for now (but still suggest to have and check updated aptly, as it should support gpg2 in master, what do you think?)

File: scripts/build/ci-deploy/Dockerfile (hunk 3 above)
Comment (fedordikarev) [reply scripts/build/ci-deploy/Dockerfile:33]: I checked docs site and for me it looks like they just renamed it.
But for now lets check with less changes and move to newer versions and naming some time later.

File: scripts/build/ci-deploy/Dockerfile (hunk 1 above)
Comment (malcolmholmes) [reply scripts/build/ci-deploy/Dockerfile]: thanks!
